--- secp256k1.orig/include/secp256k1.h	2018-12-19 17:55:12.637496000 +0300
+++ secp256k1/include/secp256k1.h	2018-12-19 18:03:10.026740204 +0300
@@ -7,6 +7,25 @@
 
 #include <stdint.h>
 
+#define secp256k1_context_create secp256k1_context_create_pp
+#define secp256k1_context_clone secp256k1_context_clone_pp
+#define secp256k1_context_destroy secp256k1_context_destroy_pp
+#define secp256k1_context_randomize secp256k1_context_randomize_pp
+
+#define secp256k1_ecdsa_verify secp256k1_ecdsa_verify_pp
+#define secp256k1_ecdsa_sign secp256k1_ecdsa_sign_pp
+#define secp256k1_ecdsa_sig_sign secp256k1_ecdsa_sig_sign_pp
+
+#define secp256k1_ec_seckey_verify secp256k1_ec_seckey_verify_pp
+#define secp256k1_ec_pubkey_create secp256k1_ec_pubkey_create_pp
+#define secp256k1_ec_privkey_tweak_add secp256k1_ec_privkey_tweak_add_pp
+#define secp256k1_ec_pubkey_tweak_add secp256k1_ec_pubkey_tweak_add_pp
+#define secp256k1_ec_privkey_tweak_mul secp256k1_ec_privkey_tweak_mul_pp
+#define secp256k1_ec_pubkey_tweak_mul secp256k1_ec_pubkey_tweak_mul_pp
+
+#define secp256k1_nonce_function_default secp256k1_nonce_function_default_pp
+#define secp256k1_nonce_function_rfc6979 secp256k1_nonce_function_rfc6979_pp
+
 # if !defined(SECP256K1_GNUC_PREREQ)
 #  if defined(__GNUC__)&&defined(__GNUC_MINOR__)
 #   define SECP256K1_GNUC_PREREQ(_maj,_min) \
